monitoring/README.md

```diff
@@ -9,23 +9,22 @@ cp .env.example .env
 ```
 * Make sure your prometheus config [file](./prometheus.yml) is updated with the metrics port (`NODE_METRICS_PORT`) of the eigenda node.
 * Make sure the eigenda container name is also set correctly in the prometheus config file. 
-You can find that in eigenda [.env](../.env) file (`MAIN_SERVICE_NAME`)
-* Make sure the location of prometheus file is correct in [.env](./.env) file
+You can find that in eigenda [.env](../.env.example) file (`MAIN_SERVICE_NAME`)
+* Make sure the location of prometheus file is correct in [.env](./.env.example) file
```

Comment:
use upper case when referring to "Prometheus"

Reply:
done